examples/prom_02_grafana_cloud/config.h

```diff
@@ -1,8 +1,8 @@
 #define WIFI_SSID     ""
 #define WIFI_PASSWORD ""
 
-#define GC_URL "prometheus-us-central1.grafana.net"
+#define GC_URL "http://prometheus-us-central1.grafana.net"
```

Comment:
This http shouldn't be there can you take this back out pls?

Reply:
Will do